src/node_file.cc

```diff
@@ -82,6 +82,12 @@ constexpr char kPathSeparator = '/';
 const char* const kPathSeparator = "\\/";
 #endif
 
+std::string Basename(const std::string& str) {
+  std::string::size_type pos = str.find_last_of(kPathSeparator);
```

Comment:
There’s a presumably rare edge case of Unix filesystems allowing `\` as a valid filename character.

Reply:
@richardlau I’m not sure I understand what you’re referring to? `kPathSeparator` is `/` on Unix

Reply:
Sorry, didn’t expand the diff upwards to see it was a conditional definition.